### ISSUE TITLE
Any plans to train transducer models?

### ISSUE THREAD

**Issue by janvainer** [k2-fsa/icefall#110]
Hi, are there any plans to experiment with models trained with a transducer loss?
 Torch 1.10 started supporting [RNNTransducer loss](https://pytorch.org/audio/0.10.0/transforms.html#torchaudio.transforms.RNNTLoss).
